Revise  animation-timeline example (#43902)
mdn/content: files/en-us/web/css/reference/properties/animation-timeline/view/index.md

#### HTML

> The HTML includes a lot of content, including two overlays, which we've hidden for brevity. In the middle of the text, we include the following:

Reviewer: You didn't hide the overlays - they appear next.

```suggestion
In the middle of the text, we include the following (note that the HTML includes a lot of content that we've hidden for brevity):
```

    scelerisque. Netus et malesuada fames ac.
  </p>
</div>
```

We also include two overlays to enable visualizing the animation range:

```html
<div class="overlay top">inset start 50%</div>
<div class="overlay bottom">inset end 10%</div>
```

#### CSS

> Most of the CSS is hidden for brevity. The styles for the `subject` and `subject-container` include:

Reviewer: This is just a preference, but I would move "Most of the CSS is hidden for brevity. " to the end and change to "The rest of the CSS is hidden for brevity. "

Author: agreed. updated. thanks.
